apps/E2E/src/common/BasePage.ts

```diff
@@ -237,31 +237,36 @@ export class BasePage {
   // Returns: UI Element
   // The Text component on each test page containing the title of that page. We can use this to determine if a test page has loaded correctly.
   get _testPage() {
+    console.error('You have to implement the method _testPage.');
```

Comment:
To provide a bit more context, let's add "Each class extending BasePage must implement it's own ___"